### ISSUE TITLE
Inferior performance for the fine-tuned object detector

### ISSUE THREAD

**Issue by fredzzhang** [fredzzhang/hicodet#3]
The performance of the fine-tuned detector (on testset) from this repository is inferior compared to the detections provided in [DRG](https://github.com/vt-vl-lab/DRG), despite the rather significant improvement from the pre-trained detector.

|Model|mAP|mRec|
|:-|:-:|:-:|
|Pre-trained on MS COCO|`32.45`|`69.75`|
|Fine-tuned on HICO-DET (DRG)|`65.12`|`86.93`|
|Fine-tuned on HICO-DET (This repo)|`46.16`|`72.27`|

At the moment, different learning rate schedules have been explored and resulted in only insignificant improvements. The reason behind the inferior performance is still unknown

**Comment by fredzzhang** [comment]
As an additional reference, [detections](https://drive.google.com/file/d/1QI1kcZJqI-ym6AGQ2swwp4CKb39uLf-4/view) on the test set provided by [Hou et al.](https://arxiv.org/abs/2007.12407) are also evaluated. The performance is shown below.

|Model|mAP|mRec|
|:-|:-:|:-:|
|Fine-tuned on HICO-DET(Hou et al.)|`45.00`|`63.98`|

Based on the print messages, 9 classes do not have any predicted examples (also reflected in the mean maximum recalls). This could suggest the object detector was not fine-tuned properly.

**Comment by fredzzhang** [comment]
To understand the performance gap between training and testing, we evaluated our fine-tuned detector on the training set and show the performance at different epochs.

|Epoch|Training mAP| Training mRec|Test mAP|Test mRec|
|:-:|:-:|:-:|:-:|:-:|
|9|`65.96`|`88.73`|`46.16`|`72.29`|
|...|
|13|`67.36`|`89.60`|`46.16`|`72.27`|
|14|`68.45`|`89.92`|`46.14`|`72.06`|

As shown in the table above, the mAP of our fine-tuned detector on the training set is comparable to the test mAP of the detections provided by the DRG repo
